docker/utils/v4r_and_openni_kinetic.sh

```diff
@@ -22,7 +22,6 @@ mkdir build && cd build
 cmake ..
 cmake --build . --target ObjectRecognizer
 cmake --build . --target RTMT
-make install
```

Comment:
please comment this code and add some text explaining why you did this. In 3 month we all forget about it and get to the same issue :)